hr_timesheet_time_type/README.rst

```diff
@@ -0,0 +1,83 @@
+======================
+Time Type in Timesheet
+======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Ftimesheet-lightgray.png?logo=github
+    :target: https://github.com/OCA/timesheet/tree/12.0_hr_timesheet_time_type/hr_timesheet_time_type
+    :alt: OCA/timesheet
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/timesheet-12-0_hr_timesheet_time_type/timesheet-12-0_hr_timesheet_time_type-hr_timesheet_time_type
```

Comment:
I think these 12.0 references in the readme will auto update to 14 by OCA bots but may want to update them since you have to change the version in the manifest anyway.